airflow/providers/microsoft/azure/log/wasb_task_handler.py

```diff
@@ -62,6 +63,7 @@ def __init__(
         **kwargs,
     ) -> None:
         super().__init__(base_log_folder, filename_template)
+        self.handler: logging.FileHandler | None = None
```

Comment:
Can you undo this change?
it's part of https://github.com/apache/airflow/pull/31855/files

Reply:
Ah - without this, type checking fails, so it seemed appropriate otherwise checks would be failing on main?

But also, https://github.com/apache/airflow/pull/31855/files has just now been merged, so this is a mute point? That PR was rebased on this one

Reply:
In hindsight/for future reference, maybe it should have been 3 PRs so checks passed in all of them?

1. One for this line in wasb_task_hander.py
2. One for the change in file_task_handler.py, rebased on 1., then rebased on main after 1. was merged
3. One for the redis task handler, rebased on 2., then rebased on main after 2. was merged?

?

Reply:
@eladkal is right, usually we try to split core and provider PR because it makes it harder for the release manager.

In cases where it is now possible to do so, then we don't split and try to resolve things a cherry picking time. This can prove difficult and require many PRs from the provider to be manually cherry picked to allow to cherry pick the PR with core change. If it's too much effort, it can postpone the release of this patch until next minor.t

Reply:
yeah I think now it should be fine with just the core as the provider code is already updated in main branch
@michalc can you update the PR to drop the `wasb_task_handler.py` change?

Reply:
@eladkal Done

Reply:
(But have deliberately not rebased on main because I _think_ it will reduce this PR to nothing?)

Reply:
ah oops
I didn't notice `airflow/utils/log/file_task_handler.py` was not removed from https://github.com/apache/airflow/pull/31855 when I merged it.

So NVM... the release manager will handle it as bulk.. :)

Reply:
Ah in that case I’ll close this PR?

Reply:
Ah I see just done